src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/DefaultComesLast/context/defaultContextConfig.xml

```diff
@@ -9,7 +9,7 @@
         <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
             <property name="file" value="${tp}/defaultContext.patch" />
             <property name="strategy" value="context" />
-            <property name="supportContextStrategyChecks" value="DefaultComesLastCheck" />
+            <property name="neverSuppressedChecks" value="DefaultComesLastCheck" />
```

Comment:
Please explain why the old property doesn't suit here?

Reply:
```
@@ -9,6 +9,8 @@ public class Test {
                 break;
             default:  // violation without filter
                 break;
+            case 3:
+                break;
         }
```
in this case, two added line will cause the violation, but them are not the child nodes of `default:  `, as shown below, so if we do not use `neverSuppressedChecks`, this violation will be suppressed.

```
   |       |   |           `--SEMI -> ; [9:21]
    |       |   |--CASE_GROUP -> CASE_GROUP [10:12]
    |       |   |   |--LITERAL_DEFAULT -> default [10:12]
    |       |   |   |   `--COLON -> : [10:19]
    |       |   |   `--SLIST -> SLIST [11:16]
    |       |   |       `--LITERAL_BREAK -> break [11:16]
    |       |   |           `--SEMI -> ; [11:21]
    |       |   |--CASE_GROUP -> CASE_GROUP [12:12]
    |       |   |   |--LITERAL_CASE -> case [12:12]
    |       |   |   |   |--EXPR -> EXPR [12:17]
    |       |   |   |   |   `--NUM_INT -> 3 [12:17]
    |       |   |   |   `--COLON -> : [12:18]
    |       |   |   `--SLIST -> SLIST [13:16]
    |       |   |       `--LITERAL_BREAK -> break [13:16]
    |       |   |           `--SEMI -> ; [13:21]
    |       |   `--RCURLY -> } [14:8]
```

Reply:
Can we use `checkNamesForContextStrategyByTokenOrParentSet` here instead of `neverSuppressedChecks`?

Reply:
@rdiachenko I try it, it not work, because it violation node is `default [10:12]`, so its parent node is `CASE_GROUP [10:12]`, it is still not enough. maybe after https://github.com/checkstyle/patch-filters/issues/163, we can use Ancestor node, then it can have some effect.